turn/src/server/mod.rs

```diff
@@ -117,6 +159,12 @@ impl Server {
                                 .await;
                             continue;
                         }
+                        Ok(Command::GetAllocationsInfo(five_tuples, tx)) => {
+                            let infos = allocation_manager.get_allocations_info(five_tuples).await;
+                            drop(tx.send(infos).await);
```

Comment:
Ca we use `let _ = tx.send(infos).await` instead of `drop`?

Reply:
Yes, we can. But as far as I know, it has the same behavior in this case. Can you explain this propose, please?

Reply:
It makes sense, it will be `let _`.

Reply:
@logist322 @k0nserv there are two `clippy` lints controlling this:
- [`clippy::let_underscore_drop`](https://rust-lang.github.io/rust-clippy/master/index.html#let_underscore_drop)
- [`clippy::drop_non_drop`](https://rust-lang.github.io/rust-clippy/master/index.html#drop_non_drop)

The whole idea to squat around them exists only if the crate has `#![warn(unused_results)]` `rustc` lint, or we're dealing with `#[must_use]` function/type.

Reply:
Right, but we don't care about the lifetime of the value returned by `tx.send(infos).await` hence `let _ = ..` to drop it immediately is fine?

Reply:
@k0nserv from this prespective (borrow check) there should be no difference between `drop()`, `let _ = ..` or just `tx.send(infos).await;`. It's more a code style issue rather than semantics.